modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -19,6 +19,7 @@ and other settings.
 
 . Create a `journald.conf` file with the required settings:
 +
+[source,terminal]
```

Comment:
I'm not sure `.conf` files should use terminal style. At best, it does no harm. At worse, it might make it less legible.